app/static/pages/edit_form.js

```diff
@@ -240,9 +251,17 @@ function SubmitData(submission)
     contentType: 'application/json',
     url:  "/json/" + collection + "/" + formId,
     data: JSON.stringify(submission.data),
-    success: schemaRecordChange
+    success: postSuccess
   })
-   .fail(function(res, statusCode, statusMsg)
+   .fail(postFail);
+  
+  function postSuccess(result)
+  {
```

Comment:
Function braces should be on same line as function declaration

Reply:
This whole file needs a proper cleanup anyway, which I plan to do as part of a future priority task, so I've left all the brackets and other formatting for now.